core/src/main/scala/org/apache/spark/status/AppStatusListener.scala

```diff
@@ -1353,4 +1363,22 @@ private[spark] class AppStatusListener(
     }
   }
 
+  /**
+   *
+   * @param processInfoEvent Miscellaneous Process Info Event
```

Comment:
I think this is pretty obvious so unless we are adding more useful description just remove the scala doc here

Reply:
removed it